handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
-| REVIEW      | The RFC is ready to be reviewed. The RFC explicitly lists who the [decider](../decisions.md#decider) is, if any [approvals](../decisions.md#approvers) are required (and by whom), and a requested timeline for those approvals.                                                                                                                    |
-| APPROVED    | All comment threads are resolved, all required approvals have been received, and the decision has been made to implement the RFC. It is in the author's best interest to avoid surprises at code review time by ensuring the RFC has a sufficient level of detail and has approval from all relevant stakeholders.                                  |
-| ABANDONED   | There are no plans to move forward with this RFC. The particular reason is communicated in the metadata section of the RFC. For example, the RFC may have failed to get the necessary approvals, it may be been superseded by another RFC, priorities may have changed, or we may not have resources to work on this RFC in the foreseeable future. |
-| IMPLEMENTED | This RFC has been implemented.                                                                                                                                                                                                                                                                                                                      |
+Our goal is for RFCs to be lightweight, low-process, and effortless to create and use. An RFC can be thought of as an asynchronous conversation. Unlike Slack, email, or other channels, using RFCs helps us because:
 
-A prose description of the status appears in the [metadata](#RFC-structure) of an RFC.
+- There’s no implicit demand for an immediate response.
+- Reviewers have time to consider and propose changes.
+- More people can collaborate at once without clashing.
+- RFCs are easily searchable and referable.
+- RFCs are retained indefinitely.
 
-## RFCs are sequentially numbered
+## When to use an RFC
 
-Each RFC is assigned a unique sequential number that appears in the title (e.g. "RFC 27: Standardizing RFCs"). This makes it easier to quickly reference specific RFCs in informal conversation (e.g. Slack, GitHub issues) and easier for readers to quickly find that document in a shared location (e.g. Google Drive). The sequential numbers also provide valuable information about the order in which RFCs were created.
+### Use an RFC when…
 
-When you are referring to an RFC with a group of people who might not be familiar with the contents of the RFC (e.g., a [weekly update](../../engineering/engineering-management.md#status-updates)), be descriptive of what the RFC does (if you have done a good job choosing a title for the RFC, then this is the title) instead of merely using the RFC number.
+- You want to drive a decision using our [formal decision-making process](../decisions.md).
```

Comment:
So I actually don't think this is a good reason to create an RFC. In the vast majority of cases, that process points to "just make the decision yourself and document it". 

```suggestion
```

Reply:
I agree in spirit here, and think we can add some stuff to make that really clear. I think there's a lot of benefit in using RFCs for the bigger, "highly cross-functional decisions"—of which we had at least one in the past few months.

```suggestion
- You need to surface and communicate around a highly cross-functional decision [formal decision-making process](../decisions.md).
```